test/integration/install.test.js

```diff
@@ -73,7 +73,7 @@ describe('snowpack install', () => {
     }
 
     // TODO: remove when ora is replaced
-    if (testName === 'error-node-builtin-unresolved') {
+    if (['dep-types-only', 'error-node-builtin-unresolved'].includes(testName)) {
```

Comment:
Argh. Ora (spinner) claims another Windows test. This is the output on Node 10, Windows:

```diff
- ✖ No ESM dependencies found!
        At least one dependency must have an ESM "module" entrypoint. You can find modern, web-ready packages at https://www.pika.dev
+ ✖ No ESM dependencies found!
```

The reason for a line missing is because Ora sometimes clears output. It’s somewhat of a race condition.

Reply:
sad! looking forward to completing #648